basic-rec-track.svg

```diff
@@ -116,7 +116,7 @@
 								</g>
 								<text font-size="8">
 									<tspan x="182" y="112">WG Decision</tspan>
-									<tspan x="288" y="112">Team’s approval</tspan>
+									<tspan x="288" y="112">Team Approval</tspan>
```

Comment:
This label overlaps almost exactly, but is shifted by two px, with the one from the `<!-- CRD to CR -->` block.

Reply:
Would you say one should be deleted (and if so, which), or the redundant overlay should be retained?